alembic/versions/967036f90ba1_gemini_model_py.py

```diff
@@ -0,0 +1,33 @@
+"""gemini_model.py
+
+Revision ID: 967036f90ba1
+Revises: 094ea67a2baf
+Create Date: 2018-02-27 11:15:29.518284
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '967036f90ba1'
+down_revision = '094ea67a2baf'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('osio_registered_repos',
+                    sa.Column('git_url', sa.String(length=255), nullable=False),
+                    sa.Column('git_sha', sa.Text, nullable=False),
+                    sa.Column('email_ids', sa.String(length=255), nullable=False),
+                    sa.Column('last_scanned_at', sa.DateTime(), nullable=True),
```

Comment:
This column should have been timestamp

Reply:
@samuzzal-choudhury, https://github.com/fabric8-analytics/fabric8-gemini-server/pull/1#discussion-diff-171189910R93. I have handled it here.